src/cascadia/TerminalSettingsEditor/GlobalAppearance.cpp

```diff
@@ -88,43 +105,22 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         // [1]:
         {
             // ManifestLanguages contains languages the app ships with.
-            //
-            // Languages is a computed list that merges the ManifestLanguages with the
-            // user's ranked list of preferred languages taken from the system settings.
-            // As is tradition the API documentation is incomplete though, as it can also
-            // contain regional language variants. If our app supports en-US, but the user
-            // has en-GB or en-DE in their system's preferred language list, Languages will
-            // contain those as well, as they're variants from a supported language. We should
-            // allow a user to select those, as regional formattings can vary significantly.
-            const std::array tagSources{
-                winrt::Windows::Globalization::ApplicationLanguages::ManifestLanguages(),
-                winrt::Windows::Globalization::ApplicationLanguages::Languages()

```

Comment:
So wait, the concern about regional variants either doesn't apply or doesn't matter how that we have to hardcode? Or, it would accidentally report too many language?

Reply:
@lhecker, you're the one who told me to take it out. Explain to Dustin?

Reply:
The latter. I was discussing this with @miniksa: `Languages()` returns regional variants - like `de-AT` - which can be important for number formatting and such. As such there's only two options here:
* filter the language list by reimplementing WinUI's language matcher (the one that says that `de-AT` and `de-DE` are compatible)
* hardcode the language list, ensuring only our languages are in there

The latter option seemed far more attractive since it's far simpler to implement. Additionally regional variants don't affect translated strings, but only generated strings. This means it's effect is limited to for instance the way we format numbers for the command palette. (Do we format numbers properly in a regional style anyways btw, or do we just use arabic numerals all the way?)